pkg/cmd/pr/merge/merge.go

```diff
@@ -192,9 +194,12 @@ func mergeRun(opts *MergeOptions) error {
 	}
 
 	isPRAlreadyMerged := pr.State == "MERGED"
-	if blocked := blockedReason(pr.MergeStateStatus); !opts.AutoMergeEnable && !isPRAlreadyMerged && blocked != "" {
-		fmt.Fprintf(opts.IO.ErrOut, "%s Pull request #%d is not mergeable: %s.\n", cs.FailureIcon(), pr.Number, blocked)
+	if !opts.AutoMergeEnable && !isPRAlreadyMerged && isBlocked(pr.MergeStateStatus, opts.Admin) {
+		fmt.Fprintf(opts.IO.ErrOut, "%s Pull request #%d is not mergeable: %s.\n", cs.FailureIcon(), pr.Number, blockedReason(pr.MergeStateStatus))
 		fmt.Fprintf(opts.IO.ErrOut, "To have the pull request merged after all the requirements have been met, add the `--auto` flag.\n")
+		if !opts.Admin && !isBlocked(pr.MergeStateStatus, true) {
```

Comment:
Instead of `!isBlocked(pr.MergeStateStatus, true)`, which might be hard to parse for a human, what do you think about a new function called `allowsAdminOverride(pr.MergeStateStatus)`? I think something like that could communicate intent better

Reply:
done